frontend/packages/ceph-storage-plugin/integration-tests/tests/installFlow.scenario.ts

```diff
@@ -0,0 +1,231 @@
+import { execSync } from 'child_process';
+import { ExpectedConditions as until, browser } from 'protractor';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import {
+  goToOperatorHub,
+  installOperator,
+  InstallCluster,
+  searchInOperatorHub,
+  uninstallOperator,
+  storageClusterLink,
+  btnCreate,
+  selectWorkerRows,
+  status,
+  operatorsPage,
+  ocsOperator,
+  ocsOperatorStatus,
+  filterInput,
+  getDataFromRowAndCol,
+  podNameFilter,
+  goToStorageClasses,
+  checkIfClusterIsReady,
+  getPodData,
+  getPodPhase,
+  getPodName,
+  waitFor,
+  testPodIsRunning,
+} from '../views/subscriptionFlow.view';
+import { NS, SECOND, MINUTE } from '../utils/consts';
+import { ocsDeploy } from '../utils/mocks';
+
+describe('Testing OCS Subscription', () => {
+  const installer = new InstallCluster(NS);
+
+  beforeAll(async () => {
+    await execSync(`echo '${ocsDeploy}' | oc apply -f -`);
+    // Let the cluster add defns
+    await browser.sleep(45 * SECOND);
+  }, MINUTE);
+
+  it(
+    'Subscribe to Operator',
+    async () => {
+      await goToOperatorHub();
+      const ocs = await searchInOperatorHub('Openshift Container Storage');
+      await click(ocs);
+      let uninstall = null;
+      try {
+        uninstall = !!(await uninstallOperator.isDisplayed());
+      } catch {
+        uninstall = false;
+      }
+      if (uninstall) {
+        // Cluster shouldn't be subscribed to ocs-operator beforehand
+        fail('Already subscribed to the operator');
```

Comment:
Does failing this here, will skip the tests written for install ahead?

Reply:
it will fail this particular test.